doc/tutorials/introduction/openvino_install/openvino_install.markdown

```diff
@@ -0,0 +1,543 @@
+    - OpenCV does not support Intel TBB, Intel Media SDK
+    - OpenCV DNN module can not use OpenVINO as computational backend
+- Build OpenCV from source code against specific version of OpenVINO. This approach solves the limitations mentioned above.
+
+# Approach #1: Install prebuilt OpenCV {#tutorial_openvino_install_install_prebuilt_opencv}
+
+## Installing on Ubuntu {#tutorial_openvino_install_installing_on_ubuntu}
+
+OpenCV is available from the default Ubuntu repositories:
+
+@code{.bash}
+sudo apt update
+sudo apt install libopencv-dev python3-opencv
+@endcode
+
+## Installing on RHEL {#tutorial_openvino_install_installing_on_rhel}
+
+OpenCV is available from the default RHEL repositories.
+
+1.  Enable CodeReady Linux Builder Repository:
+    @code{.bash}
+    subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+    @endcode
+
+2.  Install OpenCV:
+    @code{.bash}
+    sudo yum update
+    sudo yum install opencv opencv-devel opencv-python
+    @endcode
+
+## Installing on Windows {#tutorial_openvino_install_installing_on_windows}
+
+1.  Download OpenCV windows package on [the official OpenCV releases page](https://github.com/opencv/opencv/releases) Windows archive contains VC postfix, e.g.: `opencv-4.6.0-vc14_vc15.exe`
+2.  Extract the archive.
+3.  Setup environment variables by running the script `opencv\build\setup_vars_opencv4.cmd`
+
+## Installing on macOS {#tutorial_openvino_install_installing_on_macos}
+
+OpenCV could be installed via Homebrew:
+
+@code{.bash}
+brew install opencv
+@endcode
+
+# Approach #2: Build OpenCV against specific version of OpenVINO {#tutorial_openvino_install_build_opencv_against_specific_version_of_openvino}
+
+## Building on Ubuntu {#tutorial_openvino_install_building_on_ubuntu}
+
+### Prerequisites {#tutorial_openvino_install_building_on_ubuntu_prerequisites}
+
+1.  Install OpenVINO according to [the instruction](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_linux.html).
+2.  Install the following packages:
+
+    @code{.bash}
+    sudo apt-get install \
+    build-essential \
+    cmake \
+    ninja-build \
+    libgtk-3-dev \
+    libpng-dev \
+    libjpeg-dev \
+    libwebp-dev \
+    libtiff5-dev \
+    libopenexr-dev \
+    libopenblas-dev \
+    libx11-dev \
+    libavutil-dev \
+    libavcodec-dev \
+    libavformat-dev \
+    libswscale-dev \
+    libavresample-dev \
+    libtbb2 \
+    libssl-dev \
+    libva-dev \
+    libmfx-dev \
+    libgstreamer1.0-dev \
+    libgstreamer-plugins-base1.0-dev
+    @endcode
+
+### Procedure {#tutorial_openvino_install_building_on_ubuntu_procedure}
+
+1.  Copy OpenCV repository:
+    @code{.bash}
+	git clone --recurse-submodules https://github.com/opencv/opencv.git
+	@endcode
+
+2.  Create build directory and enter into it:
+    @code{.bash}
+    mkdir ~/build-opencv && cd ~/build-opencv
+    @endcode
+
+3.  Compile and install OpenCV:
+    @code{.bash}
+	cmake -G Ninja \
```

Comment:
I propose to remove non-relevant CMake options. Java support or Jasper do not affect the final result.

Reply:
I keep `BUILD_JAVA` and `BUILD_JASPER` because OpenVINO build used it. I'd prefer to keep it to minimize changes between OpenVINO build procedure and this document.

Reply:
I understand your point, but it does not scale for other platforms and use-cases. I propose to have 2 command line examples:
- Minimalistic: only OpenVINO/IE specific options needed for build. Everything else - OpenCV defaults.
- Configuration that reproduce OpenVINO behavior fully.